packages/eslint-plugin-ecmascript-compat/lib/features/es2019.js

```diff
@@ -37,34 +28,13 @@ module.exports = [
   },
   {
     ruleConfig: {
-      definition: coreRules.get('no-restricted-syntax'),
-      options: noRestrictedSyntaxPrototypeMethod('String.prototype.trimStart', 'ES2019'),
-    },
-    compatFeatures: [compatData.javascript.builtins.String.trimStart],
-    polyfill: 'String.prototype.trimStart',
-  },
-  {
-    ruleConfig: {
-      definition: coreRules.get('no-restricted-syntax'),
-      options: noRestrictedSyntaxPrototypeMethod('String.prototype.trimLeft', 'ES2019'),
-    },
-    compatFeatures: [compatData.javascript.builtins.String.trimStart], // not a mistake; trimLeft is an alias for trimStart
-    polyfill: 'String.prototype.trimLeft',
-  },
-  {
-    ruleConfig: {
-      definition: coreRules.get('no-restricted-syntax'),
-      options: noRestrictedSyntaxPrototypeMethod('String.prototype.trimEnd', 'ES2019'),
-    },
-    compatFeatures: [compatData.javascript.builtins.String.trimEnd],
-    polyfill: 'String.prototype.trimEnd',
-  },
-  {
-    ruleConfig: {
-      definition: coreRules.get('no-restricted-syntax'),
-      options: noRestrictedSyntaxPrototypeMethod('String.prototype.trimRight', 'ES2019'),
+      definition: esPlugin.rules['no-string-prototype-trimstart-trimend'],
```

Comment:
Reported https://github.com/eslint-community/eslint-plugin-es-x/issues/119

Reply:
Could keep the old way of doing these two while that gets fixed/released, but this PR is going to wait a while anyway because it needs to go as part of a major version due to the polyfill options name change.

Reply:
> Use the es-x/no-string-prototype-trimleft-trimright rule instead.
https://eslint-community.github.io/eslint-plugin-es-x/rules/no-string-prototype-trimleft-trimright.html

Reply:
Error: Rules with suggestions must set the `meta.hasSuggestions` property to `true`.

See referenced issue above